src/js/tabs/osd.js

```diff
@@ -910,7 +910,7 @@ OSD.constants = {
     { file: "default", name: "Default" },
     { file: "bold", name: "Bold" },
     { file: "large", name: "Large" },
-    { file: "extra_large", name: "Extra Large" },
+    { file: "extra_large", name: "X-Large" },
```

Comment:
What's overflowing here?

Reply:
It was a visual overflow in the UI like this: ![image](https://user-images.githubusercontent.com/28024/43486207-11b60648-950b-11e8-9672-3b9a0a71f793.png)

The change from Extra Large to X-Large was to fix it...

Reply:
I don't think that this is a good workaround:
- it will be disruptive for users, since they won't know that this is the same font, displayed under a new name;
- this obviously will break again as soon as the next font is added.

We should look into other solutions:
- make the list multi line;
- put it on a line of it's own, so it does not spill over the title;
- use a smaller font;
- make the dialog wider.

Reply:
Like this : 
![image](https://user-images.githubusercontent.com/28024/43489713-d8f748c0-9515-11e8-81f1-2963210e2338.png)

Reply:
Perhaps a select element (or similar javascript?)

Reply:
Yes, I think a drop down would be the most future proof, and in line with how multi selections are done elsewhere in the configurator. Sorry for sending you on a wild goose chase.

Reply:
@mikeller No problem - I'd rather explore options and get to the right solution (I'd only mocked those UI up using the inspector anyway). A select element sounds like the best solution - but would we include the "Open font file" option in it - as I don't really see it as a preset?

Reply:
@McGiverGim @mikeller 
Something like this - but what to do with custom font file?
![image](https://user-images.githubusercontent.com/28024/43548117-11a07f1a-95d5-11e8-9fee-0e9b720788c3.png)

This 
![image](https://user-images.githubusercontent.com/28024/43548381-a74f0e5a-95d5-11e8-9640-eabcf4b0cdde.png)

Or this
![image](https://user-images.githubusercontent.com/28024/43548638-61e8aee2-95d6-11e8-9e83-14e86c874ee0.png)

Reply:
I like mock-up B there.

Reply:
I like the second one from the three. Good job!

Reply:
@McGiverGim @fujin The mockup is now real - please test and feedback.

Reply:
Yep, my vote is for the second one too.